src/constants.js

```diff
@@ -8,7 +8,8 @@ const EIP_712_PROVIDER = 'Hermez Network'
 
 const ETHER_TOKEN_ID = 0
 
-const GAS_LIMIT = 200000
+const GAS_LIMIT_HIGH = 200000
+const GAS_LIMIT_LOW = 120000
```

Comment:
Instead of low and high, I would call these const GAS_LIMIT_DEPOSIT and GAS_LIMIT_FORCE_EXIT, I think that it makes more sense.